modules/sdk-coin-sui/src/lib/transactionBuilder.ts

```diff
@@ -3,19 +3,34 @@ import {
   BaseKey,
   BaseTransactionBuilder,
   BuildTransactionError,
-  FeeOptions,
+  ParseTransactionError,
   PublicKey as BasePublicKey,
   Signature,
   TransactionType,
 } from '@bitgo/sdk-core';
-import { Transaction } from './transaction';
+import { SuiObjectRef, Transaction } from './transaction';
 import utils from './utils';
 import BigNumber from 'bignumber.js';
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { PayTransactionKind } from './iface';
+
+export const SUI_GAS_PRICE = 1;
```

Comment:
do we need hardcode gas price here?

Reply:
I think we should. sui fullnode + SDK has this hardcoded as well, so we should follow suit. im not sure what would happen if we change the gas price to something other than 1 right now

Reply:
updated with comment stating that it should sync with [sui sdk](https://github.com/MystenLabs/sui/blob/f32877f2e40d35a008710c232e49b57aab886462/crates/sui-types/src/messages.rs#L338)